pallets/cdd-offchain-worker/src/lib.rs

```diff
@@ -206,6 +242,7 @@ impl<T: Trait> Module<T> {
     }
 }
 
+#[allow(deprecated)]
```

Comment:
Why was this added?

Reply:
I added the declaration from the staking pallet so it comes from there. Initially, I thought something from the rust side that is deprecated in the nightly substrate is using that internally but I just checked it compiles without it as well so we can safely remove it I suppose.